cognite/neat/app/ui/neat-app/src/views/WorkflowView.tsx

```diff
@@ -565,7 +565,7 @@ return (
               </Panel>
             </ReactFlow>
 
-            <Button variant="outlined" onClick={startWorkflow} sx={{ marginTop: 2, marginRight: 1 }}>Start workflow</Button>
+            <Button variant="outlined" onClick={ () => {saveWorkflow(); startWorkflow()} } sx={{ marginTop: 2, marginRight: 1 }}>Start workflow</Button>
```

Comment:
Sweet!

Reply:
it was easier than expected, but took a while to find out where and how to change this